src/Physics/XSectionIntegration/GSLXSecFunc.h

```diff
@@ -469,6 +469,30 @@ class dXSec_Log_Wrapper: public ROOT::Math::IBaseFunctionMultiDim
     double * fMaxes;
 };
 
+//_____________________________________________________________________________________
+//
+// GSL wrappers
+//
+//_____________________________________________________________________________________
+
+ class d2Xsec_dTCosth: public ROOT::Math::IBaseFunctionMultiDim
```

Comment:
Given that this function requires other functions defined in `MECUtils`, I think it is a good idea to put the wrapper there, in MECUtils.